process/scToProtocol/stakingToPeer.go

```diff
@@ -177,11 +177,14 @@ func (stp *stakingToPeer) UpdateProtocol(body *block.Body, nonce uint64) error {
 		var data []byte
 		data = stp.getStorageFromAccount(stakingSCAccount, blsPubKey)
 		// no data under key -> peer can be deleted from trie
-		if len(data) == 0 {
+		var existingAcc state.AccountHandler
+		existingAcc, err = stp.peerState.GetExistingAccount(blsPubKey)
+		if len(data) == 0 && !check.IfNil(existingAcc) && err != nil {
```

Comment:
maybe you can extract this in one variable

Reply:
Done